analyzers/its/Directory.Build.targets

```diff
@@ -20,7 +20,7 @@
       <SonarProjectType Condition="$(SonarUnknownProjectType) == 'true'">Unknown</SonarProjectType>
     </PropertyGroup>
 
-    <Error Text="Could not find '$(BinariesFolder)\SonarAnalyzer.dll'" Condition="!Exists('$(BinariesFolder)\SonarAnalyzer.dll')" />
+    <Error Text="Could not find '$(BinariesFolder)\SonarAnalyzer.CSharp\SonarAnalyzer.dll'" Condition="!Exists('$(BinariesFolder)\SonarAnalyzer.CSharp\SonarAnalyzer.dll')" />
```

Comment:
It should test CSharp DLL, when we're in CShapr directory now. We will ILMerge them one day anyway. The SonarAnalyzer.dll was a nice way how to avoid asking the same question twice